app/src/main/java/com/palone/planahead/services/alarms/AlarmsHandler.kt

```diff
@@ -0,0 +1,49 @@
+package com.palone.planahead.services.alarms
+
+import android.annotation.SuppressLint
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import com.palone.planahead.data.database.alert.Alert
+import com.palone.planahead.data.database.task.Task
+import java.time.LocalDateTime
+import java.time.OffsetDateTime
+
+class AlarmsHandler(
+    private val context: Context,
+    private val alarmManager: AlarmManager
+) {
+    private var _tasks: Map<Task, List<Alert>> = emptyMap()
+    fun inflateTasks(tasks: Map<Task, List<Alert>>) {
```

Comment:
Can't we just have tasks as `createAlarmEntries` param?

Reply:
Yes, this was a remnant of the testing phase